lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -684,10 +684,10 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
   def alert_ids(%__MODULE__{} = t), do: [t.alert.id]
 
   def temporarily_override_alert(%__MODULE__{} = t) do
-    # All screens that would show either of the following alerts
-    # will show static Evergreen content instead
-    # https://app.asana.com/0/1185117109217413/1203840359759366/f
-    t.alert.id not in ["482408", "482406"]
+    # Prevent Porter and Charles/MGH pre-fare screens from incorrectly communicating
+    # a RL alert that affects both the Ashmont and Braintree branches.
+    t.alert.id != "495152" and
```

Comment:
Should this be an `or` instead? We still want this alert to show at Ashmont.

Reply:
Alert is valid if alert ID is not 495152 AND the screen is not at Porter and Charles MGH. I think this is right

Reply:
`495152 != 495152 and "place-asmnl" not in ["place-portr", "place-chmnl"]` would return `false`, though, and we want this alert to return `true` at Ashmont. I have major end-of-week brain right now though so it's taking me longer than normal to wrap my head around this.

Reply:
Oof, yeah you're right. The name of the function threw me off, I thought it needed to return `true` to suppress the alert and `false` otherwise. Thank you, good catch.

Reply:
Another reason to give boolean-returning / predicate functions names that sound like questions. This one as it works now would be named `not_temporary_override?`

Reply:
OH ohhhh got it, thanks!!

Reply:
O yikes. I was actually right on accident. I thought this was `valid_candidate?` 😅 .

Reply:
Ok, the logic is updated. It's now equivalent to
> If we're showing the bad alert at Porter or Charles:
>   return false
> else:
>   return true